src/hotspot/share/gc/shenandoah/shenandoahMmuTracker.cpp

```diff
@@ -126,7 +126,6 @@ void ShenandoahMmuTracker::record_old_marking_increment(bool old_marking_done) {
   // No special processing for old marking
   double now = os::elapsedTime();
   double duration = now - _most_recent_timestamp;
-
```

Comment:
Is this change just to remove spurious differences from upstream openjdk/jdk repo?

Reply:
I had inserted and then removed some debugging instrumentation here.  I will restore the blank lines to "original" form.